consensus/src/network_tests.rs

```diff
@@ -295,6 +304,23 @@ impl NetworkPlayground {
         msg_copies
     }
 
+    /// Return the round of a given message
+    fn get_message_round(&self, msg: ConsensusMsg) -> Option<u64> {
+        match msg {
+            ConsensusMsg::ProposalMsg(proposal_msg) => {
+                let unboxed = *proposal_msg;
```

Comment:
I don't think you need this, rust has auto deref so you can directly do proposal_msg.proposal().round()

Reply:
Done